rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -233,44 +233,62 @@
       "$ref": "https://bib.rero.ch/schemas/documents/document_usage_and_access_policy-v0.0.1.json#/usageAndAccessPolicy"
     },
     "subjects_imported": {
-      "allOf": [
-        {
-          "title": "Subjects (imported)",
-          "description": "Subject from imported bibliographic data",
-          "form": {
-            "hide": true,
-            "navigation": {
-              "essential": false
-            },
-            "templateOptions": {
-              "cssClass": "editor-title"
-            }
+      "title": "Subjects (imported)",
+      "description": "Subject from imported bibliographic data",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "title": "Entities",
+        "description": "Topic (including genre/form), place, temporal, person, family or corporate body (including conferences). Always create a link to IdRef or GND, if possible.",
```

Comment:
"Imported topic, place, temporal, person, family or corporate body (including conferences), used for information. Do not enter data manually."